docs/package.json

```diff
@@ -82,8 +83,25 @@
   },
   "devDependencies": {
     "netlify-cli": "^2.59.0",
+    "remark-cli": "^8.0.1",
+    "remark-lint": "^7.0.1",
+    "remark-lint-no-dead-urls": "^1.1.0",
+    "remark-validate-links": "^10.0.2",
     "toml": "^3.0.0"
   },
+  "remarkConfig": {
+    "plugins": [
+      [
+        "remark-validate-links",
+        {
+          "prefix": "/docs/",
+          "lines": false
+        }
+      ],
+      "remark-lint",
+      "remark-lint-no-dead-urls"
```

Comment:
at the moment, we point users to only 2 external URLs in the docs, one of which is Rasa X's doc. I think it makes sense to check it upon every merge for now. If it becomes an issue later, we can always move it to a separate nightly check. How does that sound?